assets/src/edit-story/components/library/panes/text/textPane.js

```diff
@@ -42,9 +42,11 @@ const PRESETS = [
   {
     id: 'heading',
     title: __('Heading', 'web-stories'),
-    content: __('Heading', 'web-stories'),
+    content: `<span style="font-weight: 700">${__(
```

Comment:
Can we do a utility that does a very quick and limited (but side-effect-free) construction of content for our constants? Then if/when we switch the format slightly - that'd be easier to keep up-to-date.

Reply:
Not sure what you mean? If we move` __()` to another function, it'll not be completely side-effect free, as we have tooling depending on this syntax.

Reply:
Hmm. Never mind. I'm just slightly concerns about the text constant here (or translation) yielding no-valid HTML (e.g. someone would insert a "&" or "<"). Maybe we should juts wrap `__()` inside the `escapeHTML()`?